src/main/java/com/commercetools/project/sync/SyncerFactory.java

```diff
@@ -105,10 +106,13 @@ public static SyncerFactory of(
                               .sync(runnerNameOptionValue, isFullSync)
                               .toCompletableFuture(),
                           CustomerSyncer.of(sourceClient, targetClient, clock)
+                              .sync(runnerNameOptionValue, isFullSync)
+                              .toCompletableFuture(),
+                          ShoppingListSyncer.of(sourceClient, targetClient, clock)
                               .sync(runnerNameOptionValue, isFullSync)
                               .toCompletableFuture());
               return CompletableFuture.allOf(
-                  categoryAndInventoryAndCartDiscountAndCustomerSync.toArray(
+                  categoryAndInventoryAndCartDiscountAndCustomerAndShoppingListSync.toArray(
```

Comment:
Why did you put `ShoppingListSyncer` in this part?

Reply:
@lojzatran , I will remove this, Because we have dependency issues as Ahmet mentioned in other comment. But can you tell me what will be the issue if ShoppingListSyncer here?

Reply:
The issue might be that the process will not work :) Because of what Ahmet wrote below.

Reply:
Done. https://github.com/commercetools/commercetools-project-sync/pull/203/commits/0924e8c7a167c6ac223e871c9e9164e4c81b87ec